src/schematools/contrib/django/migrations/0012_schema_data_as_textfield.py

```diff
@@ -1,23 +0,0 @@
-# Generated by Django 3.1.12 on 2021-07-20 11:09
```

Comment:
Deleting migrations is rather bad practice and should be performed with great care.
Assuming both deleted migrations are applied to acceptance and production: we would have to manually revert them and then push current branch into master, otherwise we're risking deployment crash next time new migration is added to schematools.

Reply:
This was a temporary workaround for an issue that is now fixed. Removing the commit that did this.